appveyor.yml

```diff
@@ -8,7 +8,7 @@ environment:
 branches:
   only:
     - master
-    - /release-.*/
+    - philipvinc/in_place
```

Comment:
This should not be needed, PRs are automatically tested when new commits are pushed.

Reply:
My bad, I had used it to enable CI builds on my branch while experimenting. 
I reverted the commit.